parser_library/src/context/statement_cache.h

```diff
@@ -37,7 +37,8 @@ class statement_cache
     };
     // pair of processing format and reparsed statement
     // processing format serves as an identifier of reparsing kind
-    using cache_t = std::pair<processing::processing_form, cached_statement_t>;
+    using cache_t =
+        std::pair<std::pair<processing::processing_form, processing::operand_occurence>, cached_statement_t>;
```

Comment:
the whole processing_status might need to be used as a key...

Reply:
Now, that we are asking for specific instructions during parsing (ALIAS) you are probably right...
processing_kind probably doesn't matter, but doesn't hurt either.

Reply:
Or maybe just op_code is enough?

Reply:
parser_impl::resolve_expression is impacted by as well...